digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/TrafficSignLinearGeneratorSpec.scala

```diff
@@ -233,9 +233,18 @@ class TrafficSignLinearGeneratorSpec extends FunSuite with Matchers {
 
   test("create prohibitions values based on trafficSigns with additional Panels") {
     val prohibitionGenerator = new TestTrafficSignProhibitionGenerator()
-    val additionalPanel = Seq(AdditionalPanel(ValidMonFri.OTHvalue, "9-10","", 1, "", 99, 99, 99), AdditionalPanel(ValidSat.OTHvalue, "(11-12)","", 2, "", 99, 99, 99), AdditionalPanel(ValidMultiplePeriod.OTHvalue, "15-16 (17-18) 19-20","", 3, "", 99, 99, 99), AdditionalPanel(ValidMultiplePeriod.OTHvalue, "17-18","", 4, "", 99, 99, 99))
-    val prohibitionPeriod = Set(ValidityPeriod(9, 10, ValidityPeriodDayOfWeek.Weekday), ValidityPeriod(11, 12, ValidityPeriodDayOfWeek.Saturday), ValidityPeriod(17, 18, ValidityPeriodDayOfWeek.Sunday)
-      ,ValidityPeriod(15, 16, ValidityPeriodDayOfWeek.Weekday) ,ValidityPeriod(17, 18, ValidityPeriodDayOfWeek.Saturday) ,ValidityPeriod(19, 20, ValidityPeriodDayOfWeek.Sunday))
+    val additionalPanel = Seq(AdditionalPanel(ValidMonFri.OTHvalue, "", "9-10", 1, "", 99, 99, 99),
+                            AdditionalPanel(ValidSat.OTHvalue, "", "(11-12)",2, "", 99, 99, 99),
+                            AdditionalPanel(ValidMultiplePeriod.OTHvalue, "", "15-16 (17-18) 19-20", 3, "", 99, 99, 99),
```

Comment:
This example doesn't exist anymore.
The old sign who allows the 3 prohibitions at the same time doesn't exist on new sign law.

Reply:
Done.